.tugboat/steps/2-update.sh

```diff
@@ -10,7 +10,13 @@ echo "Updating..."
 mv .tugboat .tugboat-tmp
 #drainpipe-end
 composer install
-./vendor/bin/task sync
+
+# Backwards compatibility for old sync command
+# If sync_command doesnt exists, run sync
+if ! ./vendor/bin/task tugboat:sync; then
+  echo "tugboat:sync does not exist, running sync"
```

Comment:
This is actually checking if any error is returned? For example, if you are pulling down databases and the API key is wrong, won't the task fail?

I wonder if we should do something like this first to check if the task exists:

```
        task --dry tugboat:sync 2>/dev/null 1>/dev/null
```

And if I remember right, you get an exit code of `200` when the task doesn't exist.